sdk-workflows/src/main/java/io/dapr/workflows/runtime/WorkflowRuntime.java

```diff
@@ -25,18 +23,15 @@ public class WorkflowRuntime implements AutoCloseable {
 
   private static volatile WorkflowRuntime instance;
   private DurableTaskGrpcWorkerBuilder builder;
-  private DurableTaskGrpcWorker client;
+  private DurableTaskGrpcWorker worker;
 
-  private WorkflowRuntime(DurableTaskGrpcWorkerBuilder builder) throws IllegalStateException {
+  private WorkflowRuntime() throws IllegalStateException {
 
     if (instance != null) {
       throw new IllegalStateException("WorkflowRuntime should only be constructed once");
```

Comment:
Ooh, question - should I have an "1 and only 1" check added in mine?

Reply:
Not necessary for the client. Only here to ensure a single runtime.